### ISSUE TITLE
App no longer loading posts?

### ISSUE THREAD

**Issue by Zoomaaa** [AllanWang/Frost-for-Facebook#1963]
- [yes ] I have verified that the bug does not occur in my mobile browser (at https://touch.facebook.com/)

**Describe the bug**
last night the app asked me to relogin, after doing so the app shows a black screen with the loading bar stuck.

Tried clearing cache, and data but after logging in the problem is the same.

**Details (please provide at least the app version):**
 - App Version: 3.2.0
 - Device: samsung galaxy a31
 - Android Version: android 12 one ui 4.1

**Logs/Screenshots:**

![Screenshot_20231109-092325_Frost](https://github.com/AllanWang/Frost-for-Facebook/assets/45099318/a71c8f76-9329-4b5a-b263-7b6d56fa8fd0)

**Comment by osowiecki** [comment]
It has something to do with logging in. I was getting logged in as "Facebook" and not my profile. I can no longer log in in Frost. Other Apps from FDroid still work fine.

**Comment by freeeflyer** [comment]
Same here. Maybe it's related to Meta new adblock policy ?

**Comment by tigermick** [comment]
Hi,
Same problem here.
Black screen, no loading success (post or chat).
Frost don't ask me to log in again.

**Comment by Fredhoob** [comment]
Same for me since November 8th. I then opened Facebook on my desktop computer and had to validate a message about some kind of membership feature to remove ads that Facebook is now implementing. So that may have something to do with as @freeeflyer said.

**Comment by TheRealFlamepanther** [comment]
Encountering the same issue here

**Comment by jessep1nkm4n** [comment]
Same issue...

**Comment by biladina** [comment]
bump, same issue here..

**Comment by mormegil-cz** [comment]
Yep, this has started happening to me a few days ago and the app is unusable since. 

Even though I guess with the current pending total rewrite, there is not much chance of this being fixed, I tried debugging the app, and when the main screen starts, the following lines start sprinting in logcat, again and again… (I have shortened the URL, obviously)

```
2023-11-10 16:40:12.944  4826-4826  Frost    com.pitchedapps.frost.debug    V  web client: page finished reveal
2023-11-10 16:40:12.954  4826-4826  Frost    com.pitchedapps.frost.debug    V  web client: Url loading: https://www.facebook.com/?paipv=0&eav=foo...bar&_rdr
2023-11-10 16:40:12.954  4826-4826  Frost    com.pitchedapps.frost.debug    V  web client: Url path /
2023-11-10 16:40:12.957  4826-4826  Frost    com.pitchedapps.frost.debug    V  web client: finished https://m.facebook.com/?paipv=0&eav=foo...bar&_rdr
2023-11-10 16:40:12.957  4826-4826  Frost    com.pitchedapps.frost.debug    V  web client: page finished reveal
2023-11-10 16:40:12.977  4826-4826  Frost    com.pitchedapps.frost.debug    V  web client: Url loading: https://www.facebook.com/?paipv=0&eav=foo...bar&_rdr
2023-11-10 16:40:12.977  4826-4826  Frost    com.pitchedapps.frost.debug    V  web client: Url path /
2023-11-10 16:40:12.980  4826-4826  Frost    com.pitchedapps.frost.debug    V  web client: finished https://m.facebook.com/?paipv=0&eav=foo...bar&_rdr
2023-11-10 16:40:12.981  4826-4826  Frost    com.pitchedapps.frost.debug    V  web client: page finished reveal
```

So, that seems to be the culprit, even though I have no idea what is causing the problem (and why it started now).

**Comment by guysoft** [comment]
Having the issue here too

**Comment by guysoft** [comment]
@mormegil-cz Is there a way i can help? Assume something is bothering you that you did a thumbs down?

**Comment by mormegil-cz** [comment]
@guysoft What do you think your “Having the issue here too” brings to the issue? If you want just to “+1” the issue, use a thumb up reaction. Every user adding a “me too” comment just adds noise.

**Comment by guysoft** [comment]
The issue is tagged as unverified. I was not sure if it was considered as an issue yet. Added a thumbs up.

**Comment by atom-smasher** [comment]
Me too. I cleared the app cache. Then I cleared the app data. Then I had to login again. No data from FB is loading in any way that I can see it.

**Comment by shanemd** [comment]
I noticed a couple of things when investigating:
1. The mobile site (m.facebook.com) no longer has messenger functionally.
2. The mobile version of the messenger site (m.messenger.com) no longer has functionally. It just directs you to the app.

To see messages on Mobile, I had to request the "desktop site" in my browser.

It seems that meta has changed some things.

Edit: to clarify, I had to request the desktop site for the messenger site (messenger.com). I realized the wording was not clear.

**Comment by BradB58** [comment]
@shanemd The messenger function has not been a part of the Facebook mobile web app for years. With Frost being broken, I am now reduced to using the web app (through Brave Browser to reduce tracking) as it seems to be the only viable alternative for FB on android for now. However, it is quite obvious that FB/Meta wants to discourage us from using their web app. 
1. The toolbar at the top is not fixed. It scrolls away with everything else as you move down the page. If you want to use the bar you have to scroll all the way back to the beginning or refresh the page.
2. There is no option to allow android notifications. Most web apps beg you to allow notifications through your browser. Not so with FB unless you're using the desktop version.
3. As you noticed, there is no messenger function unless using desktop version.

We really need Frost back urgently! 😭

**Comment by kowith337** [comment]
I feels like all mobile versions are redirect and keep pointing to `m.facebook.com`, also they possibly ignore or block to visit posts or pages directly.
> anything behind `/` after `https://m.facebook.com` will be ignored and you're always redirect to main page with stories and default (algorithm'd) feed, include ads too.

Only certain pages are allowed, e.g. I've tested with `/onthisday` while use Mull browser.

Also if you try to access `m.facebook.com` while request desktop site is enabled, you're always redirect back to `www.facebook.com`, no matter what.

For now is...
- Basic site (mbasic) are always accessible regardless of devices, however, I've got limied to use sorting `?sk=...`, only `?sk=pages` one.
- Legitimate mobile browser (or spoofed UA) can use apps-like interface mobile site.
- Desktop browser are no longer to access mobile version anymore.
